operations/experimental/disable-logs-in-firehose-windows2019.yml

```diff
@@ -0,0 +1,8 @@
+---
+- type: replace
+  path: /instance_groups/name=windows2019-cell/jobs/name=loggregator_agent_windows/properties/disable_logs?
+  value: true
+
+- type: replace
+  path: /instance_groups/name=windows2019-cell/jobs/name=loggregator_agent_windows/properties/disable_logs?
```

Comment:
this appears to be exactly the same as the block above

Reply:
honestly, thinking about it right now, I'm not sure how that happened.